add personal statement post (#866)
DFE-Digital/bat-design-history: app/posts/apply-for-teacher-training/2023-03-24-merging-the-personal-statement.md

---
title: Merging the personal statement 
description: We merged the two personal statement sections into one and improved the guidance around what we say to candidates to help them write a good personal statement
date: 2023-03-24
tags:
  - personal statement
#ogImage:
#src: /apply-for-teacher-training/merged-personal-statement.png
#alt: "Screenshot of the personal statement page, with guidance text and a text box."

---

Previously, the personal statement had two sections on the application form: ‘Why you want to teach’ (600 words) and ‘Your suitability to teach a subject or age group’ (400 words). 

We merged these personal statement sections into one, and improved the guidance around what we say to candidates to help them write a good personal statement. 

> We tested this design and improved the content shown to candidates after conducting research. We released this feature, via a feature flag, on 20th March 2023.

Reviewer: Let’s put in the date it actually goes live (once we've confirmed it)?

```suggestion
We tested this design and improved the content shown to candidates after conducting research. We released this feature to candidates on 3 April 2023.
```

Author: So remove line for the time being @frankieroberto ?

Reviewer: Yeah, then add it back when we launch? Or say "We plan to..." and we'll update it post-launch?

## The issue

> Candidates found it difficult splitting up their experience and knowledge they have into separate questions and found there was a lot of overlap between the two sections. Primary applicants had particular difficulty in answering separate questions, because ‘why you want to be a teacher’ and ‘why you want to teach Primary’ are very similar. 

Reviewer: ```suggestion
Candidates found it difficult splitting up the experience and knowledge they have into separate questions and that there was a lot of overlap. Primary applicants had particular difficulty in answering separate questions, because ‘why you want to be a teacher’ and ‘why you want to teach Primary’ are very similar. 
```

We wanted to test whether our new guidance aligned with what candidates expect to write about, and whether it was easy to read and comprehend the guidance. 

We also wanted to learn more about how candidates would approach writing a personal statement, and what worries candidates about doing so. This will help determine what support we can give potential candidates in writing their personal statement. 

> ‘Personal statement’ is one of the highest reasons for rejection used by providers. In merging the personal statement sections into one, there was scope for us to refine our guidance to help candidates improve the quality of their personal statement. 

Reviewer: We could add a stat here? seems like Personal statement accounts for at least 18% of rejections? (possibly higher, as so many rejections use "other")

* their relevant skills, experience and activities 
* their subject interest 
* their understandings of why teaching is important 
* their reasons for wanting to teach 
* any relevant activities they’ve done relevant to teaching 

We kept in the link to the teacher training advisers sign-up page, as candidates found this option very useful in research. 

![Screenshot with the heading ‘Personal Statement’, followed by content saying the user should write between 500 and 1000 words for their personal statement. The content also gives the user some examples they could write about, including skills and experience with young people they have, subject interest, their understanding of why teaching is important, their reasons for wanting to train to teach, and activities that could be relevant to teaching. The content below tells the user they can speak to a Teacher Training adviser for one-on-one support and provides a link. The screen then has a large empty text box and tells the user that they have 1000 words remaining underneath the text box.](merged-personal-statement.png)

> 

Reviewer: Should we include a section about how we're rolling this out mid-cycle?

```suggestion

### Releasing this change mid-cycle

Because we released this change in the middle of a recruitment cycle, we didn't want it to disrupt any candidates who were midway through writing a personal statement using the previous 2 sections.

To avoid this, we only released the feature for new applications started on or after 3 April 2023.

If a candidate is unsuccessful with an application that used 2 personal statement sections and decided to apply again, they will see the new merged personal statement section. When this happens, we:

* mark the section as incomplete
* copy the previous 2 answers into the new single personal statement section, separated by a paragraph break
* show some content to explain what’s changed

(include screenshot?)
```

Author: Sounds good! Do you have the PNG for this screenshot by any chance?

## What we learnt in research

We tested the design with 8 pre-application candidates at different stages of the consideration journey. We conducted two moderated content testing activities:

> * Activity 1: candidates were presented with snippets from a personal statement and asked to write what they thought the information was describing 
> * Activity 2: candidates were presented with the design, and invited to highlight where they thought the information was helpful, clear or confusing

Reviewer: ```suggestion
* presenting participants with snippets from a personal statement and asking them to write what they thought the information was describing 
* showing participants the new guidance and inviting them to highlight where they thought the information was helpful, clear or confusing
```

* their reasons for wanting to teach 
* any relevant activities they’ve done relevant to teaching 

We kept in the link to the teacher training advisers sign-up page, as candidates found this option very useful in research. 

![Screenshot with the heading ‘Personal Statement’, followed by content saying the user should write between 500 and 1000 words for their personal statement. The content also gives the user some examples they could write about, including skills and experience with young people they have, subject interest, their understanding of why teaching is important, their reasons for wanting to train to teach, and activities that could be relevant to teaching. The content below tells the user they can speak to a Teacher Training adviser for one-on-one support and provides a link. The screen then has a large empty text box and tells the user that they have 1000 words remaining underneath the text box.](merged-personal-statement.png)

### Releasing this change mid-cycle

Because we released this change in the middle of a recruitment cycle, we didn't want it to disrupt any candidates who were midway through writing a personal statement using the previous 2 sections.

> To avoid this, we only released the feature for new applications started on or after 3 April 2023.

Reviewer 2: Should this be in a new paragraph or part of the one above? Might just need a space in between the 2 paragraphs if they are separate.

> If a candidate is unsuccessful with an application that used 2 personal statement sections and decided to apply again, they will see the new merged personal statement section. When this happens, we:

Reviewer 2: Same comment as above, might just need a space here if it's a new paragraph as it looks a bit odd on the review app.

To avoid this, we only released the feature for new applications started on or after 3 April 2023.
If a candidate is unsuccessful with an application that used 2 personal statement sections and decided to apply again, they will see the new merged personal statement section. When this happens, we:
* mark the section as incomplete
* copy the previous 2 answers into the new single personal statement section, separated by a paragraph break
* show some content to explain what’s changed

## What we learnt in research

We tested the design with 8 pre-application candidates at different stages of the consideration journey. We conducted two moderated content testing activities:

* presenting participants with snippets from a personal statement and asking them to write what they thought the information was describing 
* showing participants the new guidance and inviting them to highlight where they thought the information was helpful, clear or confusing

We found that there were no major usability or comprehension issues. The guidance met candidates’ expectations about what to write about, and no guidance was missing for them. 

> Candidates were able to think of examples for almost all the bullet points and felt comfortable with the 1000 word limit. Candidates were able to think of examples for almost all the bullet points and our word choices largely made sense to them. 

Author: Will bullet point it
